scheduler/src/cook/mesos/mesos_mock.clj

```diff
@@ -292,9 +292,10 @@
           resources' (subtract-resources available-resources requested-resources)
           tasks (map #(assoc % :launched-time (t/now)) tasks)
           task->runtime-ms (-> state :config :task->runtime-ms)
+          runtime-multiplier (get-in host [:attributes :runtime-multiplier] 1)
           new-time-task-id-pairs (map (fn [{:keys [task-id] :as task}]
-                                        [(-> (t/now) 
-                                             (t/plus (t/millis (task->runtime-ms task)))) 
+                                        [(t/plus (t/now)
+                                                 (-> task task->runtime-ms (* runtime-multiplier) t/millis))
```

Comment:
Can we move `(t/plus (t/now))` to the end of the threading macro instead of having it outside?